pkgs/utils/src/log.zig

```diff
@@ -4,7 +4,7 @@ const datetime = @import("datetime");
 
 // having activeLevel non comptime and dynamic allows us env based logging and even a keystroke activated one
 // on a running client, may be can be revised later
-pub fn compTimeLog(comptime scope: LoggerScope, activeLevel: std.log.Level, comptime level: std.log.Level, comptime fmt: []const u8, args: anytype) void {
+pub fn compTimeLog(comptime scope: LoggerScope, activeLevel: std.log.Level, comptime level: std.log.Level, comptime fmt: []const u8, args: anytype, file: ?std.fs.File) void {
```

Comment:
also add the fileActiveLevel in the last argument and change it to a params/struct so that stderr and log fiile can be run at different active log levels

for e.g. we might just want info logs on screen and full default logs in the file